NSS-Day-Cohort-64/rare-python-server-donkey-puppets
# Added subscribe to user functionality

Added functionality for subscribing to a user.

To test:
git fetch --all
git checkout subscribe-to-user
go to postman and put in this url: http://localhost:8088/subscriptions
give an example raw body to send:
{
    "follower_id": 2,
    "author_id": 2,
    "created_on": "2023-08-06T11:15:25"
}
hit send and check your SQL table to be sure the subscription was added.
